packages/pluggableWidgets/video-player-native/package.json

```diff
@@ -20,11 +20,14 @@
   },
   "dependencies": {
     "@mendix/piw-native-utils-internal": "^1.0.0",
-    "react-native-video": "5.0.2"
+    "react-native-system-navigation-bar": "^1.0.2",
```

Comment:
I think it's best to fix this version, to avoid any unexpected changes

Reply:
done